src/universal/conf/storage.conf

```diff
@@ -34,16 +34,17 @@ mantis {
   pruning {
     # Pruning mode that the application will use.
     #
-    # - archive: No pruning is performed
-    # - basic: reference count based pruning
+    # - archive:  No pruning is performed
+    # - basic:    reference count based pruning
+    # - inmemory: reference count inmemory pruning
     #
-    # After changing, please delete previous db before starting the client:
+    # After changing, please delete previous db before starting the client
     #
-    # mode = "basic"
+    # mode = "archive"

```

Comment:
For what chains it will be set up as archive with this new config encoding ?
Should not each of new `eth.conf`, `etc.conf` files have different prunning settings ? 
As for current config the settings where
inmemory - ETH
basic - everything else

Reply:
It's handled now by separate launcher script (in `src/universal/bin/mantis-eth`) for ETH which sets these settings. To keep things rather simple and not make revolution in code I wanted to avoid loading different configuration files which override whole app settings.